File: ELEC881_Copilot/src/Function Name Masked/Java script (JS)/q10-non-overlapping-intervals.js
var abc = function(intervals) {
    intervals.sort((a, b) => a[1] - b[1]);
    var count = 0;
    var end = -Infinity;
    for (var i = 0; i < intervals.length; i++) {
        if (intervals[i][0] >= end) {
            end = intervals[i][1];
        } else {
            count++;
        }
    }
    return count;
};

File: ELEC881_Copilot/src/Function Name Masked/Python/q10-non-overlapping-intervals.py
class abc(object):
    def abc1(self, intervals):
        if not intervals:
            return 0
        intervals.sort(key=lambda x: x[1])
        count = 1
        end = intervals[0][1]
        for i in range(1, len(intervals)):
            if intervals[i][0] >= end:
                count += 1
                end = intervals[i][1]
        return len(intervals) - count